src/app/nextversion/nextversion_test.go

```diff
@@ -318,10 +318,48 @@ dependencies:
   to: 1.0.0
 			`),
 		},
+	} {
+		tc := tc
+		//nolint:paralleltest // urfave/cli cannot be tested concurrently.
+		t.Run(tc.name, func(t *testing.T) {
+			repoDir := repoWithTags(t, tc.tags...)
+
+			app := app.App()
+
+			yamlPath := path.Join(repoDir, "changelog.yaml")
+			yamlFile, err := os.Create(yamlPath)
+			if err != nil {
+				t.Fatalf("Error creating yaml for test: %v", err)
+			}
+			_, _ = yamlFile.WriteString(tc.yaml)
+			_ = yamlFile.Close()
+
+			buf := &strings.Builder{}
+			app.Writer = buf
+
+			err = app.Run(strings.Fields(fmt.Sprintf("rt -yaml %s next-version -git-root %s %s", yamlPath, repoDir, tc.args)))
+			if err != nil {
+				t.Fatalf("Error running app: %v", err)
+			}
+
+			if actual := buf.String(); actual != tc.expected+"\n" {
+				t.Fatalf("Expected %q, got %q", tc.expected, actual)
+			}
+		})
+	}
+}
+
+//nolint:paralleltest // urfave/cli cannot be tested concurrently.
+func TestNextVersion_Fails(t *testing.T) {
+	for _, tc := range []struct {
+		name     string
+		yaml     string
+		args     string
+		expected string
```

Comment:
Nit: we don't need a string expected output in this set of tests at the moment, do we?

Reply:
Good catch!